internal/handler/handler.go

```diff
@@ -15,13 +15,8 @@ import (
 )
 
 var (
-	positionsRe      = regexp.MustCompile(`[\\/]positions\?limit=(\d+)&offset=(\d+)$`)
-	positionRe       = regexp.MustCompile(`[\\/]position[\\/](\S+)$`)
-	createPositionRe = regexp.MustCompile(`[\\/]position$`)
-
-	employeesRe      = regexp.MustCompile(`[\\/]employees\?limit=(\d+)&offset=(\d+)$`)
-	employeeRe       = regexp.MustCompile(`[\\/]employee[\\/](\S+)$`)
-	createEmployeeRe = regexp.MustCompile(`[\\/]employee$`)
+	posRegex = regexp.MustCompile(`[\\/]position[\\/](\S+)$`)
+	empRegex = regexp.MustCompile(`[\\/]employee[\\/](\S+)$`)
```

Comment:
Why do you need these regexp if you are using gorilla router?

Reply:
+